bridges/prometheus/producer_test.go

```diff
@@ -229,6 +229,44 @@ func TestProduce(t *testing.T) {
 				},
 			}},
 		},
+		{
+			name: "histogram cumulative values to non-cumulative",
+			testFn: func(reg *prometheus.Registry) {
+				metric := prometheus.NewHistogram(prometheus.HistogramOpts{
+					Name: "test_histogram_metric",
+					Help: "A histogram metric for testing",
+					ConstLabels: prometheus.Labels(map[string]string{
+						"foo": "bar",
+					}),
+				})
+				reg.MustRegister(metric)
+				metric.Observe(0.01)
+			},
+			expected: []metricdata.ScopeMetrics{{
+				Scope: instrumentation.Scope{
+					Name: scopeName,
+				},
+				Metrics: []metricdata.Metrics{
+					{
+						Name:        "test_histogram_metric",
+						Description: "A histogram metric for testing",
+						Data: metricdata.Histogram[float64]{
+							Temporality: metricdata.CumulativeTemporality,
```

Comment:
Shouldn't this be delta temporality?

Reply:
No, prometheus always produces cumulative metrics. This is just changing how bucket counts are represented.  OTel 's bucket count is the number that fell into that specific bucket.  Promethues' bucket count is the number that fell into that bucket **or below**.